app/main/views/briefs.py

```diff
@@ -204,6 +207,113 @@ def update_brief_status(brief_id, action):
     return jsonify(briefs=brief.serialize()), 200
 
 
+@main.route('/briefs/<int:brief_id>/award', methods=['POST'])
+def award_pending_brief_response(brief_id):
+    json_payload = get_json_from_request()
+    updater_json = validate_and_return_updater_request()
+
+    brief = Brief.query.filter(
+        Brief.id == brief_id
+    ).first_or_404()
+
+    if not brief.status == 'closed':
+        abort(400, "Brief is not closed")
+
+    brief_responses = BriefResponse.query.filter(
+        BriefResponse.brief_id == brief.id,
+        BriefResponse.status != 'draft'
+    )
+    # Check that BriefResponse in POST data is associated with this brief
+    if not json_payload['brief_response_id'] in [b.id for b in brief_responses]:
+        abort(400, "BriefResponse cannot be awarded for this Brief")
+
+    # Find any existing pending awarded BriefResponse
+    audit_events = []
+    pending_awarded_responses = list(filter(lambda x: x.status == 'pending-awarded', brief_responses))
+    if pending_awarded_responses:
+        # Reset the existing awarded BriefResponse (should only be one)
+        pending_award = pending_awarded_responses[0]
+        pending_award.award_details = {}
+        db.session.add(pending_award)
+        audit_events.append(
+            AuditEvent(
+                audit_type=AuditTypes.update_brief_response,
+                user=updater_json['updated_by'],
+                data={
+                    'briefId': brief.id,
+                    'briefResponseId': pending_award.id,
+                    'briefResponseAwardedValue': False
+                },
+                db_object=pending_award
+            )
+        )
+
+    # Set new awarded BriefResponse
+    brief_response = list(filter(lambda x: x.id == json_payload['brief_response_id'], brief_responses))[0]
+    brief_response.award_details = {'pending': True}
+    audit_events.append(
+        AuditEvent(
+            audit_type=AuditTypes.update_brief_response,
+            user=updater_json['updated_by'],
+            data={
+                'briefId': brief.id,
+                'briefResponseId': brief_response.id,
+                'briefResponseAwardedValue': True
+            },
+            db_object=brief_response
+        )
+    )
+
+    db.session.add_all([brief_response] + audit_events)
+    db.session.commit()
+
+    return jsonify(briefs=brief.serialize()), 200
+
+
+@main.route('/briefs/<int:brief_id>/award/<int:brief_response_id>/contract-details', methods=['POST'])
+def award_brief_details(brief_id, brief_response_id):
+    json_payload = get_json_from_request()
+    updater_json = validate_and_return_updater_request()
+
+    brief = Brief.query.filter(
+        Brief.id == brief_id
+    ).first_or_404()
+
+    brief_response = BriefResponse.query.filter(
+        BriefResponse.id == brief_response_id
+    ).first_or_404()
+    if not brief_response.award_details.get('pending'):
+        abort(400, "Cannot save details for this brief")
```

Comment:
As Ben I think mentioned elsewhere, it would be good to say _why_ in the message, even if we don't think it's likely / possible to actually come up in real life. (Because developers are users too :) )